lib/Params.ml

```diff
@@ -689,38 +690,39 @@ let box_pattern_docked (c : Conf.t) ~ctx ~space_around ~pat opn cls k =
     | Ast.Exp {pexp_desc= Pexp_let ({pvbs_bindings; _}, _, _); _}, _
       when List.exists pvbs_bindings ~f:(fun b -> phys_equal b.pvb_pat pat)
       ->
-        (-4, 0)
+        (-4 - ext_length, 0)
```

Comment:
You might not need to propagate `ext_length` as the information is in the context in the corresponding binding's `pvb_attributes` field.

Reply:
I will try it

Reply:
Its done